manifests/init.pp

```diff
@@ -147,4 +180,7 @@
   # Dependency added for Grafana_plugins to ensure it runs at the
   # correct time.
   Class['grafana::config'] -> Grafana_Plugin <| |> ~> Class['grafana::service']
+
+  create_resources('grafana_ldap_server', $ldap_servers)
```

Comment:
If you're interested, here's a more modern approach to `create_resources()`. No need to change this though.

https://github.com/sensu/sensu-puppet/blob/master/manifests/resources.pp#L78-L82